### DISCUSSION TITLE
[Question] Are there plans to support Remix.run like Next.js

### DISCUSSION THREAD

**Discussion by palcisto** [gluestack/gluestack-ui#445] [Ideas]
Forgive my naiveness but, are there plans to support SSR with Remix.run much like there is for Next.js. As I'm beginning a new project with Remix however, it doesn't seem to me that Gluestack-UI works out of the box with Remix as I assume it would need an adapter much like you've created for Next.js.

**Comment by sanketsahu** [comment]
With the rising popularity of Remix, we are considering this. We will add updates to this thread as we plan this. Thank you!

**Comment by palcisto** [comment]
Awesome! If I can be of any help with any low-hanging fruit, please let me know.

**Comment by jacargentina** [comment]
Almost one year soon, any updates on this?

Maybe some Remix user/developer which is already using gluestack-ui with some custom hack?

**Comment by ankit-tailor** [comment]
Hi Everyone, We have added remix support to our roadmap. We'll share the updates as we start working on it.

**Reply by palcisto** [reply]
That's great news! Is the roadmap available for us to see or is there any timeline on when we can roughly expect to see a preview of Remix support?

**Reply by FezVrasta** [reply]
@ankit-tailor is there any update on Remix support?
